projects/1.20-fabric/assets/debugify/debugify/lang/zh_cn.json

```
@@ -0,0 +1,79 @@
+{
+	"debugify.name": "Debugify",
+	"debugify.basic": "基础修复",
+	"debugify.gameplay": "游戏修复",
+	"debugify.gameplay.warning": "警告：此类别可能包含对他人来说不公平的修复功能！",
+	"debugify.gameplay.enable_in_multiplayer": "在多人游戏中启用",
+	"debugify.fix.enabled": "已修复",
+	"debugify.fix.disabled": "未修复",
+	"debugify.fix.unavailable": "不可用",
+	"debugify.error.conflict": "%s修复与模组“%s”不兼容",
+	"debugify.error.os": "%s仅适用于%s",
+	"debugify.error.mixin_error": "加载%s失败，详细信息请查看日志。",
+	"debugify.error.mixin_error.text": "错误",
+	"debugify.os.windows": "Windows",
+	"debugify.os.macos": "macOS",
+	"debugify.os.linux": "Linux",
+	"debugify.os.solaris": "Solaris",
+	"debugify.misc": "杂项",
+	"debugify.misc.default_disabled": "默认禁用新的修复功能",
+	"debugify.misc.default_disabled.description": "默认禁用新的错误修复功能。",
+	"debugify.env.client": "客户端",
+	"debugify.env.client.desc": "只适用于客户端的修复功能。",
+	"debugify.env.server": "主要",
+	"debugify.env.server.desc": "适用于客户端和服务器与它们之间的共同代码的修复功能。",
+	"debugify.no_yacl.title": "未安装YetAnotherConfigLib！",
+	"debugify.no_yacl.description": "要使用GUI配置Debugify，你必须安装%s，这是此功能的前置。你也可以在储存于“%s”的JSON文件中修改配置。",
+
+	"debugify.mc_237493.header": "遥测已由 Debugify 关闭",
+	"debugify.mc_237493.line": "你在遥测选项上选择了“%s”，因此Debugify正在阻止与Mojang的遥测服务的所有连接。这是由Debugify提供的MC-237493修复。",
+	"debugify.mc_237493.tooltip.off": "“%s”完全禁用了与Mojang遥测服务的任何连接。（Debugify MC-237493）",
+	"debugify.mc_237493.tooltip.minimal": "“%s”只包括Mojang认为需要的信息。",
+	"debugify.mc_237493.tooltip.all": "“%s”包括所有的最少和可选遥测。",
+
+	"debugify.fix_explanation.mc-55347": "断开连接后清除屏幕标题。",
+	"debugify.fix_explanation.mc-79545": "在渲染时将经验等级限制为整型限制。",
+	"debugify.fix_explanation.mc-93384": "将溺水粒子生成位置提升到眼睛的高度。",
+	"debugify.fix_explanation.mc-108948": "问题是客户端运行的运算逻辑与服务器不同。当没有人控制船时，客户端不会在运行船的物理模拟，导致服务端与客户端不同步。这个修复程序告诉客户端只运行船的物理运算。",
+	"debugify.fix_explanation.mc-111516": "修改数学acos运算以确保结果不超过1.0。",
+	"debugify.fix_explanation.mc-112730": "如果信标和其他方块实体已添加到全局列表中，则会阻止它们添加到本地方块实体渲染器列表中。",
+	"debugify.fix_explanation.mc-122477": "防止在编辑栏（例如聊天栏）初始化后进行前两次键盘轮询。",
+	"debugify.fix_explanation.mc-123739": "按标识符对配方条目进行排序。",
+	"debugify.fix_explanation.mc-143474": "重生后生成新的本地玩家时，复制选定的物品槽。",
+	"debugify.fix_explanation.mc-159163": "从实体数据包处理中删除姿势数据。",
+	"debugify.fix_effect.mc-159163": "这将阻止客户端与服务器同步姿势数据，完全由客户端来决定姿势。",
+	"debugify.fix_explanation.mc-162253": "在加载区块时停止客户端光照更新。",
+	"debugify.fix_explanation.mc-165381": "当客户端丢弃物品时取消破坏方块",
+	"debugify.fix_explanation.mc_176559": "在检查是否应重置破坏方块时与比较物品变化时不考虑物品耐久。",
+	"debugify.fix_explanation.mc-197260": "覆盖传递给盔甲架渲染器的光照等级，最大值为：盔甲架下方的方块、盔甲架底部、盔甲架顶部和盔甲架上方的方块。",
```

原文：_Overrides the light level passed to the armour stand renderer with **a maximum of:**_ the block below the armor stand, bottom of the armor stand, top of the armor stand and the block above of the armor stand.
```suggestion
	"debugify.fix_explanation.mc-197260": "将传递给盔甲架渲染器的光照等级设置为以下四处的最大值：盔甲架下方的方块、盔甲架底部、盔甲架顶部和盔甲架上方的方块。",
```